### ISSUE TITLE
Website accessibility test for boston.gov

### ISSUE THREAD

**Issue by subaha-cob** [CityOfBoston/Iterators#209]
The goal is to test the site to make sure all the changes made to adjust and add certain accessibility feature did not break the page and/or page content for other users. Iterators are testing for the changes or updates made for accessibility only. So they will concentrate on that.

What we are testing for:

- [x] Text Contrast - New grey color now uses (#58585b) instead of #d2d2d2
- [x] Charles blue background - text on Charles blue background is now #28a7df
- [ ] Focus state should match hover state
- [ ] All form elements including buttons, search, input fields must be focusable and clickable
- [x] All external links should open in a new tab/window
- [ ] Sidebar images are now real images instead of background images
- [x] New back to top functionality
- [x] Replaced some HTML elements that are not semantic
- [x] Components, images, or sections "ID" tag attribute should be unique. I removed duplicates for a lot of elements. Test to make sure links, buttons, tabs, accordions, menus, SVG files, modals, among others still works.
- [x] Replaced a few background images to become actual images because they were more content than decorative. test to make sure images are not broken on the website
- [x] Speed. Test to see if pages still loads as fast 

Thank you!

**Comment by padora-cob** [comment]
- [ ]  Focus state should match hover state

This is looking good to me, for the most part, after checking a number of pages and content types. As I navigate via the keyboard everything is changing to the hover state as expected.

I may have found one more component though: https://d8-uat.boston.gov/affordable-housing-boston#types-of-housing

** I was hovering over the "Metrolist" link at the time and keyboard focused on "Income Restricted Housing Guide".
<img width="1802" alt="image" src="https://user-images.githubusercontent.com/62249548/117485418-58edb400-af36-11eb-90ff-bca0b14a65b5.png">

**Comment by reillyzlab** [comment]
Reilly test report back:

- [ ] Text Contrast - New grey color now uses (#58585b) instead of #d2d2d2 - didn't test but see that Alex did
 
- [ ] Charles blue background - text on Charles blue background is now #28a7df - not sure how you want this tested so please advise if you'd like me to test? I can inspect but not sure where to test on the site.
 
- [ ] Focus state should match hover state
Didn't pass because of:
- Grid of guides has a focus state but hover isn't showing
- Events component is not showing a hover state
- Not sure if too early/not relevant but back to top does not have a hover state
  
- [x] All form elements including buttons, search, input fields must be focusable and clickable
- [x] All external links should open in a new tab/window
- [ ] Sidebar images are now real images instead of background images - not sure how to test this one
 
- [ ] New back to top functionality
- Functionally working but didn't show up on test page until TEST GROUP OF LINKS - LIST WITHOUT DESCRIPTION. Seems like it should be showing up earlier?
- Doesn't have hover state but should.
 
- [x] Replaced some HTML elements that are not semantic - didn't know how to test so didn't test 
- [ ] Components, images, or sections "ID" tag attribute should be unique. I removed duplicates for a lot of elements. Test to make sure links, buttons, tabs, accordions, menus, SVG files, modals, among others still works. - don't have ANDI installed and think need that to test so didn't test
- [x] Replaced a few background images to become actual images because they were more content than decorative. test to make sure images are not broken on the website
- [ ] Speed. Test to see if pages still loads as fast -- UAT seems to be running slower than production:
<img width="706" alt="Screen Shot 2021-05-07 at 10 23 32 AM" src="https://user-images.githubusercontent.com/20731897/117486404-7020a780-af1e-11eb-8378-d78bd9b5e98a.png">
<img width="708" alt="Screen Shot 2021-05-07 at 10 24 20 AM" src="https://user-images.githubusercontent.com/20731897/117486415-744cc500-af1e-11eb-94bd-bf810ba1b380.png">

**Comment by padora-cob** [comment]
Both Metrolist and Building Housing functionality seems to be working as expected. 

> - [x]  Text Contrast - New grey color now uses (#58585b) instead of #d2d2d2
> - [ ]  Charles blue background - text on Charles blue background is now #28a7df
> - [x]  Focus state should match hover state
> - [x]  All form elements including buttons, search, input fields must be focusable and clickable
> - [x]  All external links should open in a new tab/window
> - [ ]  Sidebar images are now real images instead of background images
> - [x]  New back to top functionality
> - [ ]  Replaced some HTML elements that are not semantic
> - [ ]  Components, images, or sections "ID" tag attribute should be unique. I removed duplicates for a lot of elements. Test to make sure links, buttons, tabs, accordions, menus, SVG files, modals, among others still works.
> - [ ]  Replaced a few background images to become actual images because they were more content than decorative. test to make sure images are not broken on the website
> - [x]  Speed. Test to see if pages still loads as fast
<img width="864" alt="image" src="https://user-images.githubusercontent.com/62249548/117486702-f5648600-af37-11eb-966d-181b8c955cc3.png">
<img width="892" alt="image" src="https://user-images.githubusercontent.com/62249548/117486818-247af780-af38-11eb-81da-324c1af97651.png">

**Comment by subaha-cob** [comment]
Testing is over. Closing ticket.
